### PR TITLE
CP-27819: Switch to Visual Studio 2017

### PR DESCRIPTION
**Pull request by AaronRobson**
Upgrading solutions and projects including the latest ToolsVersion and tidying up after the upgrades.

### CODE REVIEW

File: splash/splash.vcxproj [hunk 1]
```diff
@@ -117,6 +117,12 @@
   <ItemGroup>
     <ResourceCompile Include="splash.rc" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (kc284) [inline splash/splash.vcxproj]: Is this needed?

File: XenAdmin/XenAdmin.csproj [hunk 2]
```diff
@@ -14,11 +14,6 @@
     <SignManifests>false</SignManifests>
     <ManifestCertificateThumbprint>4C1DD393E361B8BBCFA1E1D09539C446F88C0A11</ManifestCertificateThumbprint>
```
Comment (kc284) [inline XenAdmin/XenAdmin.csproj]: Not related to this PR, but I've just noticed the tags ManifestCertificateThumbprint and ManifestTimestampUrl which exist in some of the solution's projects and are ancient; we've been using `mk\sign.bat` for signing for a while now.

File: splash/splash.vcxproj (hunk 1 above)
Comment (AaronRobson) [reply splash/splash.vcxproj]: You are right, this does not appear to be required for it to build and run, I'm not sure why the solution/project updater would add it. I will remove it.

File: XenAdmin/XenAdmin.csproj (hunk 2 above)
Comment (AaronRobson) [reply XenAdmin/XenAdmin.csproj]: I see, would removing both these nodes from each project they are present in be your recommended course of action?

File: XenAdmin.sln [hunk 3]
```diff
@@ -13,9 +13,6 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "xva_verify", "xva_verify\xva_verify.csproj", "{2A70D7E7-EAB2-4C36-B3F4-85B79D2384B5}"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "splash", "splash\splash.vcxproj", "{AFB19C9D-DD63-478B-A4A3-8452CBD0B9AB}"
-	ProjectSection(ProjectDependencies) = postProject
```
Comment (kc284) [inline XenAdmin.sln]: I've also noticed this bit has been removed.

Comment (AaronRobson) [reply XenAdmin.sln]: What was this bit doing before it was removed? Ensuring that another project identified by this GUID is compiles after the splash project perhaps?

Comment (kc284) [reply XenAdmin.sln]: I did some more googling, it seems this was needed previously to keep track of dependencies and after the upgrade it's moved to the proejcte references - which explains now the change in splash.vcxproj. Ok, my bad, let's leave this here and revert 6af64589103676ed57380eb3b2c2261a375aa779.

**Pull request merged**